src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -107,14 +107,14 @@ private void handleCommandInputChanged() {
             // process result of the command
             commandTextField.setText("");
             logger.info("Result: " + commandResult.feedbackToUser);
-            raise(new NewResultAvailableEvent(commandResult.feedbackToUser));
+            raise(new NewResultAvailableEvent(commandResult.feedbackToUser, false));
```

Comment:
Avoid using 'true' or 'false. Notice that you have used it whenever you use this method. Try to refactor the code.